packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -2,10 +2,11 @@ import { ITooltipStyleProps, ITooltipStyles } from './Tooltip.types';
 import { AnimationClassNames } from '../../Styling';
 
 export const getStyles = (props: ITooltipStyleProps): ITooltipStyles => {
-  const { className, gapSpace = 0, maxWidth, theme } = props;
+  const { className, beakHeight = 16, gapSpace = 0, maxWidth, theme } = props;
   const { palette, fonts, effects } = theme;
 
-  const tooltipGapSpace = -15 - gapSpace;
+  // The -4 is added so that hidden area doesn't overlap with the Tooltip target.
+  const tooltipGapSpace = -(beakHeight + gapSpace - 4);
```

Comment:
For comparison, this is what it looks like without the `-4` at the end:

![image](https://user-images.githubusercontent.com/7798177/61985827-f5dc9880-afbf-11e9-85a1-9c1ebd02a1c6.png)

And this is what it looks like once we add the `-4`:

![image](https://user-images.githubusercontent.com/7798177/61985838-068d0e80-afc0-11e9-94d5-cbd65efd1b08.png)

Reply:
had added a top level comment about this but putting in code as well; I feel like the original problem was caused by putting an arbitrary "15" hardcoded in the styles, and now we're sticking this "4" in with seemingly no reason other than to "make it work". Can we get to the bottom of why it's needed?